src/generate/peripheral.rs

```diff
@@ -1235,14 +1239,19 @@ fn expand_register(
                     &description,
                 );
                 let mut accessors = Vec::with_capacity((array_info.dim + 1) as _);
+                let first_name = svd::array::names(info, array_info).next().unwrap();
+                let note = (array_info.indexes().next().unwrap() != "0").then(||
+                    format!("<div class=\"warning\">`n` is number of {0} in array. `n == 0` corresponds to `{first_name}` {0}.</div>", "register")
```

Comment:
```suggestion
                    format!("<div class=\"warning\">`n` is the index of {0} in the array. `n == 0` corresponds to `{first_name}` {0}.</div>", "register")
```

i think this can be worded better, maybe this?